coding/logging.rst

```diff
@@ -0,0 +1,231 @@
+#######
+Logging
+#######
+
+This page provides guidance to developers for using logging with :lmod:`lsst.log`  in the science pipeline code base. 
+Developers are encouraged to insert log messages whenever and wherever they might be useful, with appropriate component names and levels.
+
+.. _logger-names:
+
+Logger Names
+============
+Logger names should generally start with the fully qualified name of the module/file containing the logger, without the leading ``lsst.``.
+If the logger is saved as a variable in a class, it is often appropriate to name the logger after the class.
+Logger names use ``.`` as component separators, not ``::``, even in C++.
+Some example logger names are ``afw.image.MaskedImage``, ``meas.algorithms.starSelector``, etc.
+
+
+Basic Usage in Python
+=====================
+
+The basic Python interface of :lmod:`lsst.log` is made to be somewhat similar to Python logging.
+The simplest way to log is:
+
+.. code-block:: python
+
+   import lsst.log
+   lsst.log.info('Some information during normal operation')
+   lsst.log.warn('Here is a warning!')
+
+
+The example above logs to the root logger.
```

Comment:
You could say "This example logs [...]"

Separate this sentence and "A better practice..." in two separate paragraphs.